packages/core-data/src/resolvers.js

```diff
@@ -961,3 +961,26 @@ export const getRevision =
 			dispatch.receiveRevisions( kind, name, recordKey, record, query );
 		}
 	};
+
+/**
+ * Requests a specific post type options from the REST API.
+ *
+ * @param {string} postType Post type slug.
+ */
+export const getRegisteredPostMeta =
+	( postType ) =>
+	async ( { select, dispatch } ) => {
+		try {
+			const restBase = select.getPostType( postType )?.rest_base;
+			const options = await apiFetch( {
+				path: `wp/v2/${ restBase }/?context=edit`,
```

Comment:
This needs to take into account the `rest_namespace`, it might not be `/wp/v2`.

Reply:
The new selector/actions should probably be private to avoid deprecations in case we change minds after a release or two.